server/audit_math/minerva.py

```diff
@@ -1,11 +1,111 @@
+# pylint: disable=invalid-name
 """
-An implementation of Minerva
+Library for performing a Minerva-style ballot polling risk-limiting audit,
+as described by Zagórski et al https://arxiv.org/abs/2008.02315
+
+Note that this library works for one contest at a time, as if each contest being
+targeted is being audited completely independently.
+
+TODO: ensure the no-losers case is handled
+TODO: if necessary pull out risks for individual contests
+
 """
-# TODO: remove the following once we are using the arguments
-# pylint: disable=unused-argument
-from typing import Dict, Tuple, Optional
+from decimal import Decimal
+from collections import defaultdict
+import logging
+from typing import List, Dict, Tuple, Optional
 
+from athena.audit import Audit  # type: ignore
```

Comment:
For the sake of clarity, could you import this as `AthenaAudit` or something?

Reply:
Good idea.